src/OrchardCore.Modules/OrchardCore.Taxonomies/Drivers/TermPartContentDriver.cs

```diff
@@ -41,11 +44,17 @@ public override Task<IDisplayResult> DisplayAsync(ContentItem model, BuildDispla
             {
                 return Task.FromResult<IDisplayResult>(Initialize<TermPartViewModel>("TermPart", async m =>
                 {
+                    // Provide the route data for the term to the pager so it generates the pager links
+                    // using the Autoroute path instead of the jsonPath.
+                    var termContentItemMetadata = await _contentManager.PopulateAspectAsync<ContentItemMetadata>(model);
```

Comment:
I probably shouldn't assume the term content item has the AutoroutePart. I thought about adding

if (model.Has<AutoroutePart>())
{
    // Do the route data stuff
}

but I couldn't access the AutoroutePart in this project.

Reply:
Please try https://github.com/OrchardCMS/OrchardCore/issues/12870#issuecomment-1343119061.

Reply:
It's been so long; I will try to refresh my memory and take another look.

Reply:
Thank you!